data/json/items/armor.json

```diff
@@ -2909,6 +2909,28 @@
     "flags": [ "WATER_FRIENDLY", "SUN_GLASSES" ]
   },
   {
+    "id": "eclise_glasses",
```

Comment:
There is a `p` missing. The game should have complained that the new recipe uses the undefined item "eclipse_glasses".